README.md

```diff
@@ -108,7 +108,8 @@ Use HELP for help.
 Curl over port 8082
 
 ```sh
-curl -L -X GET 'localhost:8082/v1/health' \
+curl -L -X GET 'localhost:8082/v1/keyspaces' \
```

Comment:
Should it not be documented / linked somewhere how to get this AUTH_TOKEN?
Maybe as part of this example?

Reply:
This will be replaced with full docs soon.